cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -92,7 +104,7 @@ func (dc *DownstreamController) syncPod() {
 			if err := dc.messageLayer.Send(*msg); err != nil {
 				klog.Warningf("send message failed with error: %s, operation: %s, resource: %s", err, msg.GetOperation(), msg.GetResource())
 			} else {
-				klog.V(4).Infof("send message successfully, operation: %s, resource: %s", msg.GetOperation(), msg.GetResource())
+				klog.V(4).Infof("message sent successfully, operation: %s, resource: %s", msg.GetOperation(), msg.GetResource())
```

Comment:
"message sent successfully, operation: %s, resource: %s" this is repeated multiple times. consider putting it into a common string var.

Reply:
updated.